소수찾기.js

```diff
@@ -0,0 +1,30 @@
+function solution(numbers) {
+    const number = numbers.split('').sort((a, b) => b - a);  
+    const max = Number(number.join('')); 
+    const sosuNum = sosu(max);
+    let answer = 0;  
+    
+    for (let i = 2; i <= max; i++) { 
+        if (sosuNum[i] === false) continue; 
+        let temp = i.toString().split('');
+        
+        for (let el of number) { 
+            let idx = temp.indexOf(el); 
+            if (idx > -1) temp.splice(idx, 1); 
+        }
+        if(temp.length === 0) answer++ 
+    } 
+    return answer; 
+}  
+
+function sosu(N) { 
+    const arr = []; 
+    for (let i = 2; i <= N; i++) { 
+        if (arr[i] === false) continue
+        for (let k = i + i; k <= N ; k += i) { 
+            arr[k] = false; 
```

Comment:
위의 조건문에서 let i 로 선언해 준 뒤 다음 조건문은 j로 작성해 주는게 좋을 것 같아요! 
변수 선언시 i -> j -> k 순서로 작성해주는게 컨벤션으로 알고있어요 :)

```suggestion
        for (let j = i + i; j <= N ; j += i) { 
            arr[j] = false; 
```